ci/build-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
```

Comment:
I think using something like "_/usr/bin/env bash_" in shebang is a better idea since Debian and probably many other distributions have bash in _/bin_ instead of _/usr/bin_.